vtkext/private/module/vtkF3DImguiActor.cxx

```diff
@@ -359,12 +364,18 @@ void vtkF3DImguiActor::RenderCheatSheet()
       ImVec2 currentLine = ImGui::CalcTextSize(line.c_str());
 
       winWidth = std::max(winWidth, currentLine.x);
+      textHeight += ImGui::GetTextLineHeightWithSpacing();
+      ;
     }
   }
+
   winWidth += 2.f * ImGui::GetStyle().WindowPadding.x + ImGui::GetStyle().ScrollbarSize;
+  textHeight += 2.f * ImGui::GetStyle().WindowPadding.y;
+
+  winTop = std::max(marginTopBottom, (viewport->WorkSize.y - textHeight) * 0.5f);
```

Comment:
```suggestion
  const float winTop = std::max(marginTopBottom, (viewport->WorkSize.y - textHeight) * 0.5f);
```
last nitpick: this variable is declared above and never read until you override it here so might as well declare it here and make it const while you're at it.